packages/ui/src/1_atoms/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { ReactNode, FC } from 'react';
+
+import classNames from 'classnames';
+
+import { Heading } from '../Heading/Heading';
+import { HeadingType } from '../Heading/Heading.types';
+import { Icon } from '../Icon/Icon';
+import { IconNames } from '../Icon/Icon.types';
+import styles from './Accordion.module.css';
+
+export interface IAccordionProps {
+  label: string | ReactNode;
+  children: ReactNode;
+  className?: string;
+  disabled?: boolean;
+  open?: boolean;
+  onClick?: (toOpen: boolean) => void;
+  dataLayoutId?: string;
+}
+
+export const Accordion: FC<IAccordionProps> = ({
+  label,
+  children,
+  className,
+  disabled = false,
+  open = false,
+  onClick,
+  dataLayoutId,
+}) => {
+  return (
+    <div className={classNames(styles.accordion, className)}>
+      <div
+        className={classNames(styles.label, {
+          [styles.disabled]: disabled,
+        })}
+        onClick={() => !disabled && onClick?.(!open)}
+        data-layout-id={dataLayoutId}
+      >
+        <>
+          {typeof label === 'string' ? (
+            <Heading type={HeadingType.h3}>{label}</Heading>
+          ) : (
+            label
+          )}
+        </>
+        <div className={styles.arrow}>
+          <Icon
+            icon={open ? IconNames.ARROW_DOWN : IconNames.ARROW_UP}
```

Comment:
maybe would look like better if icon would rotate with animation when state changes, example can be taken from dropdown component

Reply:
nice recommendation, updated